configs/purebred.hs

```diff
@@ -67,3 +67,12 @@ main = purebred $ tweak defaultConfig where
     . set (confComposeView . cvSendMailCmd) writeMailtoFile
     . set (confFileBrowserView . fbHomePath) getCurrentDirectory
     . set (confComposeView . cvIdentities) fromMail
+    . over confTheme (applyAttrMappings myColoredTags)
+
+myColoredTags :: [(AttrName, Attr)]
+myColoredTags =
+  [ ("inbox", fg cyan)
+  , ("archive", fg cyan)
+  , ("signed", fg green)
+  , ("attachment", fg brightRed)
+  ]
```

Comment:
what happens if the tag does not match any of these?  What colour/style does it get?

IMO we want to construct the `AttrName` for the tag name as something like `"tag" <> tagName` where `tagName` is `"inbox"`, `"signed"`, etc.  This would allow to specify the default style for a tag name (with attr name `"tag"`) as well as overriding the style for specific tags.

Reply:
The default is returned. I need to specify it when I create the attribute map. In `Config.Main`:
```Haskell
solarizedDark :: A.AttrMap
solarizedDark =
    A.attrMap
        V.defAttr  -- <-- default
        [ (listAttr, V.brightBlue `on` V.brightBlack)
--  and so on
```

Reply:
Yeah but isn't that a default style for *everything*, not just tags?

Reply:
Yes. That is correct.